.github/workflows/snap.yml

```diff
@@ -0,0 +1,60 @@
+name: Snap
+
+on:
+  pull_request:
+    branches: [main]
+
+env:
+  SNAP_NAME: chisel
+
```

Comment:
```suggestion
```

I could be missing something, but I can't see where this is used?

Reply:
I guess this is something that will only be used for the "promotion" steps of this workflow.

Reply:
Indeed. @cjdcordeiro is right. Cleaned it up, will add it when I need it.